setup.py

```diff
@@ -104,6 +104,7 @@
         use_2to3=True,
         python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
         install_requires=[
+            "requests>=2.9.1",
```

Comment:
What is the reason to pin at this specific version please?

Reply:
It's not a pin, just a greater than. It's what was in the earlier PR :-)

We could probably leave it empty; I doubt anyone is trying to use a requests library > 3 years old (and if they are, they're probably on python 2.6 or 3.3 or something, and we'll have other problems anyway!)